src/docfx/cli/Docfx.cs

```diff
@@ -110,6 +110,7 @@ private static (string command, string workingDirectory, CommandLineOptions opti
                     // Restore command
                     syntax.DefineCommand("restore", ref command, "Restores dependencies before build.");
                     syntax.DefineOption("o|output", ref options.Output, "Output directory in which to place restore log.");
+                    syntax.DefineOption("use-cache", ref options.UseCache, "ONly fetch latest content if it does not exist or read from disk cache.");
```

Comment:
This IMO could be the default behavior when executing `docfx build`. Rebuilding contents fast from the command line is the most frequent ask.

The same pattern could be found for `dotnet run`, `npm run`, etc. These commands don't automatically fetch latest dependencies each time, if you absolutely want the latest dependency, explicitly execute `dotnet restore` and `npm restore`. For us it is the same story. `docfx build` alone ensures the docset is build and build runs fast. To force latest dependency, execute `docfx restore` explicitly.

See `dotnet run --no-cache --no-dependencies` as example patterns. The reason I didn't use `--no-cache` term for docfx is that we have other cache files like github user cache, it could be called `--no-restore-cache`

Reply:
Basically I did not change the behavior of `docfx build` in this PR, do not fetch anything during `build`. `Fetch` only happens during `docfx restore`, for now seems like the default behavior of `docfx restore` is to always fetch the latest. That is why I added `--uses-cache` for `restore` here

Reply:
I see, the --no-cache flag looks more like --force flag described here:https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore?tabs=netcore2x

Also this need to update docs-pipeline as this is a behavior difference.

Reply:
Right, I will open another breaking PR paired with docs-pipeline change